bootstrap4/templates/bootstrap4/inputTypes/autocomplete/autocomplete.html

```diff
@@ -1,7 +1,10 @@
 <template name="afAutocomplete_bootstrap4">
-  <div class="autocomplete dropdown">
-    <input type="text" value="{{this.label}}" {{this.visibleAtts}} />
-    <div class="dropdown-menu"></div>
-  </div>
-  <input type="hidden" value="{{this.value}}" {{this.atts}}/>
+        <input type="text" value="{{this.label}}" {{attsPlusFormControlClass this.visibleAtts}} />
```

Comment:
I see, passing the visible attributes (no name!) explicitly...but check that the 'form-control' class is not already added elsewhere...in backing autocomplete.js.

Reply:
Yes, because the default autocomplete should be theme agnostic while the theme package should add the respective classes